.golangci.yml

```diff
@@ -0,0 +1,23 @@
+run:
+  timeout: 10m
+
+linters-settings:
+  lll:
+    tab-width: 4
```

Comment:
Please add the `    line-length: 120` as well so we can see what our default should be. Also, this will avoid problems if the default changes at a future date.

Reply:
Done